seaborn/rcmod.py

```diff
@@ -460,6 +433,35 @@ def set_context(context=None, font_scale=1, rc=None):
     mpl.rcParams.update(context_object)
 
 
+class _StyleOrContext(dict):
```

Comment:
I think a better name for this would be `_RCAesthetics`; less constraining for the future.